builtin/logical/pki/cert_util.go

```diff
@@ -139,7 +139,7 @@ func fetchCAInfoByIssuerId(ctx context.Context, b *backend, req *logical.Request
 		LeafNotAfterBehavior: entry.LeafNotAfterBehavior,
 	}
 
-	entries, err := getURLs(ctx, req)
+	entries, err := getURLs(sc.Context, sc.Storage)
```

Comment:
as a get, this would also make sense to change to a sc call.

Reply:
I had it as such, then discussion with Steve kinda turned me against it. Maybe with the caching layer it makes sense, I dunno.

Reply:
Happy to be considered wrong about this, seems like I might be in the minority opinion on this topic.

Reply:
For getURLs in particular, if we do AIA info, this is going to change so I'm not inclined to do this now.